2-mandatory/3-financial-times.js

```diff
@@ -5,7 +5,15 @@
     Implement the function below, which will return a new array containing only article titles which will fit.
 */
 function potentialHeadlines(allArticleTitles) {
-    // TODO
+  // TODO
+  let articleTitles = [];
+
+  for (i = 0; i < allArticleTitles.length; i++) {
+    if (allArticleTitles[i].length <= 65) {
+      articleTitles.push(allArticleTitles[i]);
+    }
+  }
+  return articleTitles;
```

Comment:
It is right, the shortest way is using filter() like:
`function potentialHeadlines(allArticleTitles) {
  return allArticleTitles.filter((element) => element.length <= 65);
}`